src/jit/block.h

```diff
@@ -281,7 +283,9 @@ struct BasicBlock
             next->bbPrev = this; 
     }
 
-    unsigned            bbNum;      // the block's number
+    unsigned            bbNum : 31;      // the block's number
+    unsigned            bbIsLIR : 1;     // Indicates whether or not this is an LIR block
+
```

Comment:
Is this because there are no BBF flags left?

This will make all bbNum access slower.

Why don't you steal BBF_FORWARD_SWITCH? that could be under #if defined(LEGACY_BACKEND) && defined(FEATURE_STACK_FP_X87)